samples/bluetooth/fast_pair/locator_tag/Kconfig.sysbuild

```diff
@@ -10,6 +10,29 @@ config BT_FAST_PAIR
 config APP_DFU
 	bool "Enable MCUmgr OTA DFU over Bluetooth"
 	default y
+
+if APP_DFU
+
+choice APP_DFU_BACKEND
```

Comment:
Why is any of this needed? You can simply check for CONFIG_BOOTLOADER_MCUBOOT in the application code, or presumably something for SUIT

Reply:
We will remove this choice. However, there is no flag similar to the `BOOTLOADER_MCUBOOT` so the nRF54H SoC Kconfig option will be used as a dependency for now. The issue has already been flagged to the Charon team and hopefully, they will create such an option.

Reply:
but isn't that what you are doing already below, checking for SUIT? `depends on SUIT`

Reply:
The `SUIT` Kconfig is available in the application image space but not in the sysbuild space. Actually, the SUIT configuration at the sysbuild level is not needed at all. I created this artificial configuration for SUIT to have the same user configuration for both DFU solutions: MCUboot and SUIT

Reply:
Ah yes, missed that this was sysbuild, so the lines here can go completely and have it handled in just the application level Kconfigs